runtime/gc_glue_java/ObjectModelDelegate.cpp

```diff
@@ -84,3 +84,24 @@ GC_ObjectModelDelegate::calculateObjectDetailsForCopy(MM_EnvironmentBase *env, M
 	*hotFieldAlignmentDescriptor = clazz->instanceHotFieldDescription;
 }
 #endif /* defined(OMR_GC_MODRON_SCAVENGER) */
+
+void
+GC_ObjectModelDelegate::initializeMinimumSizeObject(MM_EnvironmentBase *env, uint8_t *allocAddr)
```

Comment:
We can consider reusing initializeAllocation method (populate AllocateInitialization parameter properly with j/l/o etc), but let's get with this first and consider simplification later (the population might not be much simpler than the body of this method)
Also let's use void * (not uint8_t *), to align with initializeAllocation's parmeter void *allocatedBytes

Reply:
btw, when we go into SATB for gencon, this initialization will be more complex anyway (since the object will go into Tenure, age bits should be set)